### PR TITLE
Feature/update form mock data

### PR DESCRIPTION
**Pull request by TeddyAndersson**
This PR updates the mockdata in order to have more than one Step in a Form. Some adjustments for passing props had to be done.

### CODE REVIEW

File: source/components/organisms/Step/Step.js [hunk 1]
```diff
@@ -44,6 +48,7 @@ function Step({
   onFieldChange,
   isBackBtnVisible,
 }) {
+  console.log(fields);
```
Comment (dannil76) [inline source/components/organisms/Step/Step.js:51]: should remove development related code

File: source/containers/Form/Form.js [hunk 2]
```diff
@@ -31,12 +31,14 @@ function Form({ startAt, steps, firstName, onClose }) {
       <FormStepper active={formState.counter}>
         {formState.steps.map(step => (
           <Step
+            banner={{ iconSrc: step.icon }}
```
Comment (dannil76) [inline source/containers/Form/Form.js:34]: perhaps step obj could be destructed at row 32?

File: source/assets/mock/form-case-ekb.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ const formEkbMockData = {
           footer: {},
           text: {
             colors: {
-              primary: '#00213f',
+              primary: '#00213F',
```
Comment (dannil76) [inline source/assets/mock/form-case-ekb.js:27]: should be lowercase

Comment (TeddyAndersson) [reply source/assets/mock/form-case-ekb.js:27]: Why lowercase? We are handling all other hexcodes with uppercase?

Comment (dannil76) [reply source/assets/mock/form-case-ekb.js:27]: just do as I tell you

**Pull request merged**